README.md

```diff
@@ -57,28 +59,36 @@ config = DataConfig(
 
 2. Use an eval algorithm with a custom dataset
 ```
-eval_algo = get_eval_algorithm("toxicity", ToxicityConfig())
+eval_algo = Toxicity(ToxicityConfig())
 eval_output = eval_algo.evaluate(model=model_runner, dataset_config=config)
 ```
 
-*Please refer to [code documentation](http://aws.github.io/fmeval) and
-[examples]((https://github.com/aws/fmeval/tree/main/examples)) for understanding other details around the usage of
+*Please refer to the [developer guide](https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-foundation-model-evaluate-auto.html) and
+[examples](https://github.com/aws/fmeval/tree/main/examples) for more details around the usage of
 eval algorithms.*
 
 ## Development
 
-### Setup
-Once a virtual environment is set up with python3.10, run the following command to install all dependencies:
+### Setup and the use of `devtool`
+Once you have set up virtual environment with python3.10, run the following command to install `devtool`:
```

Comment:
Suggestion: For the sake of clarity: change this to "... run the following command to setup the development environment ..." and remove the sentence "Then you can install the library with all dependencies ...". We anyways mention `./devtool all` one sentence later (the sentence starting with "Before submitting a PR ...")